app/build.gradle.kts

```diff
@@ -80,7 +80,16 @@ android {
             useLegacyPackaging = true
         }
         resources {
-            excludes += "**"
```

Comment:
Yes, in my too many tests with it was ` excludes += "**"`, Android Studio won't pack META-INF and addon folder into apk, and after my check, we need to reset what did https://github.com/bmax121/APatch/commit/8c5c34e35d9836aa8aaebff6bcab9b3588868ea2 do to make it can pack to apk

Reply:
> Yes, in my too many tests with it was `excludes += "**"`, Android Studio won't pack META-INF and addon folder into apk, and after my check, we need to reset what [8c5c34e](https://github.com/bmax121/APatch/commit/8c5c34e35d9836aa8aaebff6bcab9b3588868ea2) did to make it can pack to apk

Why not add script to assets folder?

Reply:
Already move them to assets folder